widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -1062,7 +1073,7 @@ class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defSty
     }
 
     override fun onFileOpenClick(file: AttachmentFile) {
-        val contentUri = FileHelper.getContentUri(file, context)
+        val contentUri = context.getContentUriCompat(file.fileName)
```

Comment:
In my opinion, it's more logical to keep this functionality in FileHelper rather than as extension function for Context.

Reply:
fixed - [cf6761a](https://github.com/salemove/android-sdk-widgets/pull/561/commits/cf6761ad6ab810bf0926f5c4b8f350825ca881f8)